tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -2577,46 +2492,59 @@ TEST_F(OpConverterTest, ConvertBinary) {
         "both input as constant at: my_add");
   }
 
+  using OpFunc = std::function<NodeDef(DataType)>;
+  std::map<std::string, std::pair<OpFunc, std::vector<float>>> op_map;
```

Comment:
Can we use a more specific name, such as op_to_test_info?

Reply:
Renamed to op_test_info